wmh.go

```diff
@@ -7,14 +7,16 @@ import (
 	"gonum.org/v1/gonum/stat/distuv"
 )
 
+const maxUint16 = 65536
```

Comment:
this is incorrect, the actual maximum value is one less than this
see math.MaxUint16

Reply:
The name is a misnomer.

Reply:
what would you name it? twoPower16?